test/javascripts/acceptance/post-admin-menu-test.js

```diff
@@ -7,6 +7,7 @@ QUnit.test("Enter as a anon user", async assert => {
   await click(".show-more-actions");
 
   assert.ok(exists("#topic"), "The topic was rendered");
+  assert.ok(exists("#post_1 .post-controls .edit"), "The edit button was not rendered");
```

Comment:
I didn't see any JS changed but the tests were changed?

Reply:
https://github.com/discourse/discourse/blob/032703e6e21df3754a429f839e364a4396d8371b/lib/guardian/post_guardian.rb#L164-L166

An extra condition was added to the can_edit_post? method, which ends up being used by the PostSerializer. So I added an extra assertion here to ensure the button was being rendered as expected, based on the value from the serializer.

Reply:
That is cool, just noting that it worked before this commit.

Reply:
Absolutely - just making sure I didn't introduce any regressions.